quickstarts/aws/amazon-workmail/config.yml

```diff
@@ -0,0 +1,31 @@
+slug: amazon-workmail
+description: |-
+  ## What is Amazon WorkMail?
+  Amazon WorkMail is a secure, managed business email and calendar service with support for existing desktop and mobile email client applications. Amazon WorkMail gives users the ability to seamlessly access their email, contacts, and calendars using the client application of their choice, including Microsoft Outlook, native iOS and Android email applications, any client application supporting the IMAP protocol, or directly through a web browser.
+  ### Get started!
+  Start monitoring Amazon WorkMail by connecting Amazon Web Services (AWS) to New Relic! Take a look at our documentation for information on how to set up this integration.
+  ### More info
+  Check out the [documentation](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-workmail-monitoring-integration/) to learn more about New Relic monitoring for Amazon WorkMail.
+summary: |-
+  Monitor Amazon WorkMail by connecting AWS to New Relic
+icon: logo.png
+level: New Relic
+authors:
+  - New Relic
+title: Amazon WorkMail
+documentation:
+  - name: Amazon WorkMail installation docs
+    description: |
+      Monitor Amazon WorkMail by connecting AWS to New Relic.
+    url: >-
+      https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-workmail-monitoring-integration/
+keywords:
+  - aws
+  - amazon web services
+  - WorkMail
+dashboards:
+  - amazon-workmail
+dataSourceIds:
```

Comment:
you're not referencing the `amazon-workmail` data source. did you want to reference it? If you're not using the data source you created please remove it.

Reply:
@brammerl , Addressed, I made the required changes.